system/src/control/config.cpp

```diff
@@ -55,8 +60,203 @@ namespace control {
 
 namespace config {
 
+namespace {
+
 using namespace particle::control::common;
 
+struct SecurityModeChangeContext {
+    char clientNonce[32];
```

Comment:
This will eventually contain a start time for the 1 minute lifetime of the client nonce, right?

Reply:
Yes :+1: